app/build.gradle

```diff
@@ -30,4 +30,5 @@ dependencies {
     compile 'com.android.support:cardview-v7:23.1.1'
     compile 'com.android.support:recyclerview-v7:23.1.1'
     compile 'com.jakewharton:butterknife:7.0.1'
+    compile 'com.android.support:support-v4:23.1.1'
```

Comment:
Any reason why you use support-v4 instead of support-v13? As this app has min sdk 15, support-v13 should be good to go.